templates/fine-tune-llm_v2/cookbooks/modifying_hyperparameters/README.md

```diff
@@ -0,0 +1,99 @@
+# Modifying hyperparameters
+
+**⏱️ Time to complete**: 15 minutes
+
+This guide will focus on how you can customize your fine-tuning run by modifying the various hyperparameters configurable. Make sure you've read the [basic fine-tuning guide](../../README.md) for better context. 
+
+We provide a number of options to configure via the training YAML. To view the full set of customizable parameters, head over to the [fine-tuning config API reference](https://docs.anyscale.com/reference/finetuning-config-api). Below, we'll take a closer look at some of the important hyperparameters.
+
+
+## GPU resources
+
+Configuring GPU resources to be used is one of the most important pre-requisities for training. There are two fields in our YAML that are relevant:
+
+```yaml
+num_devices: 16 # number of GPUs 
+worker_resources:
+  accelerator_type:A10G: 0.001 # specifies GPU type available, and a minimum allocation per worker
+```
+
+Internally, our fine-tuning code will launch Ray workers with each being allocated one GPU. The cluster will be auto-scaled if needed to meet the requirements. The different GPU types you can specify can depend on the specific Anyscale Cloud. The value you specify for the accelerator type (0.001 here) does not matter much, as long as it is non-zero (so that each worker is allocated a GPU) and less than or equal to 1 (so that the requested number of GPUs is the same as `num_devices`).
+
+## Learning rate
+
+There are two entities of interest here: the actual learning rate value itself and the particular learning rate scheduler you use. The parameters you can control in the YAML are below: 
+
+```yaml
+learning_rate: 1e-4
+lr_scheduler_type: cosine
+num_warmup_steps: 10
+```
+
+In the above config, the training run would use a cosine learning rate schedule (the default) with an initial warmup of 10 steps (the default). The peak learning rate would be 1e-4 (the value specified). 
+
+We support both `'linear'` and `'cosine'` schedules. The learning rate schedules have been plotted below with `num_warmup_steps=10` and for 1000 training steps.
+
+<p align="center">
```

Comment:
The image doesn't get rendered here because the path makes sense once this PR is merged into `main`.